pai-management/node-list-example.yaml

```diff
@@ -16,20 +16,25 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 
-machinelist:
+machine-list:
```

Comment:
Where is this file used?

Reply:
machine-list will be used in pai_machine() related commands, please refer to [paictl readme](https://github.com/Microsoft/pai/blob/management-dev/pai-management/doc/paictl.md)

Reply:
I will do some change this week, to make this style compatible with k8sClusterMangement tool